backend/src/services/api/userService.ts

```diff
@@ -0,0 +1,12 @@
+import * as userRepository from '../../data/repositories/userRepository';
```

Comment:
lets do the file name only on the `kebab notation` backend/src/services/api/user-service/user-service.service.ts

Reply:
Lets move user service just into `backend/src/services/user-service/user-service.service.ts`